plugins/CoreHome/stylesheets/layout.less

```
@@ -461,6 +461,34 @@ nav {
   }
 }
 
+.badge-menu-item-container {
+  line-height: inherit;
+  position: relative;
+}
+.badge-menu-item {
+  position: absolute;
+  top: 12px;
+  left: 23px;
+  line-height: 1.25;
+  background-color: rgba(212, 19, 13, 1);
+  color: #fff;
+  border-radius: 8px;
+  padding: 4px 6px;
+  font-size: 9px;
+  font-weight: bold;
+}
+
+.whatisnew-changelist {
+  overflow: scroll;
+  max-height:500px;
+  margin-bottom:10px;
```

The right border of the changes boxes currently collapse with the displayed scroll bar. Adding a small padding might let it look a bit better maybe.
```suggestion
  margin-bottom:10px;
  padding-right: 5px;
```

---

Makes sense when there are long change descriptions :+1: